index.html

```diff
@@ -1644,6 +1644,147 @@ <h4>Background</h4>
+		in her line of work. Since she can’t use the internet, most of her preferred methods of communication
+		are unavailable. When she meets a new person she wishes to communicate with, she establishes a
+		direct secure connection between her device and theirs by having them scan a QR code to pass on her
+		DID and start sharing messages secured by the keys referenced in her DID Document, which is served from
+		her device. She can maintain these secure connections whether she has access to the internet or not.
+		Whenever Samira is working on a new campaign or wants to raise awareness about issues affecting her
+		local region, she is able to securely communicate with all relevant parties, regardless of
+		the device, platform, or network infrastructure they are using.
+        </p>
+        <p>
+          At one point, Samira sees a friend of hers, Ahmed, who’s part of a local organization campaigning against unfair
+		labor practices in local agriculture. Samira wants to support the efforts of this group and get
+		involved, but she worries that publicly allying with the farm workers will negatively affect her
+		ability to act as a neutral party in labor negotiations. By creating a unique DID
+		for herself in the group at very little cost, she can engage in multi-party secure communication
+		with all of the members of Ahmed’s organization. Messages are secured via mutual authentication
+		between Samira and each of the other members. Using these connections, Samira is able to have private
+		and confidential communications with the other members at any time, without fear of censorship or
+		surveillance. Members of the organization are able to have confidence that they’re talking to
+		Samira, and she can verify who she is communicating with. Samira can use the same identifier and
+		connection with the group to share credentials that prove she’s a humanitarian worker and attest
+		to her history.
+        </p>
+        <p>
+          Eventually, Samira loses her phone. To make sure no one impersonates her, she rotates the keys on all
+		of the DIDs she is using for communication. Rather than losing access to all of her existing
+		connections, Samira is able to update her service endpoints and routing capabilities to make
+		sure she continues to receive messages securely on the devices she still owns. When she gets a new
+		phone, she can add new keys to her DID Document to continue exchanging secure messages with her
+		connections without interruption. Should Samira want to change the DIDs she’s using (for instance, because
+		she’s changing DID method providers), she can send a message to her connections notifying them
+		that she’s deprecating her old DID and providing her new DID with authentic digital signatures.
+		Samira is able to manage all of her secure communications across her devices, platforms,
+		and applications in a consistent way, thanks to the security features provided by DIDs.
```

Comment:
whitespace and line lengths
```suggestion
Eventually, Samira loses her phone. To make sure no one impersonates her, 
she rotates the keys on all of the DIDs she is using for communication. 
Rather than losing access to all of her existing connections, Samira is 
able to update her service endpoints and routing capabilities to make sure 
she continues to receive messages securely on the devices she still owns. 
When she gets a new phone, she can add new keys to her DID Document to 
continue exchanging secure messages with her connections without 
interruption. Should Samira want to change the DIDs she’s using (for 
instance, because she’s changing DID method providers), she can send a 
message to her connections notifying them that she’s deprecating her 
old DID and providing her new DID with authentic digital signatures. 
Samira is able to manage all of her secure communications across her 
devices, platforms, and applications in a consistent way, thanks to 
the security features provided by DIDs.
```